tsfile/src/main/java/org/apache/iotdb/tsfile/fileSystem/fileInputFactory/HDFSInputFactory.java

```diff
@@ -27,13 +28,21 @@
 public class HDFSInputFactory implements FileInputFactory {
 
   private static final Logger logger = LoggerFactory.getLogger(HDFSInputFactory.class);
-  private static Class<?> clazz;
+  private static Constructor constructor;
+
+  static {
+    try {
+      Class<?> clazz = Class.forName("org.apache.iotdb.tsfile.fileSystem.HDFSInput");
+      constructor = clazz.getConstructor(String.class);
+    } catch (ClassNotFoundException | NoSuchMethodException e) {
+      e.printStackTrace();
```

Comment:
Do not use stack print, use logger instead.
But on this occasion, I would suggest you throw a runtime exception (the best to be customized) because this is a fatal error.